deepchem/models/torch_models/layers.py

```diff
@@ -3056,3 +3056,87 @@ def forward(self, inputs: torch.Tensor):
         atom_enbeddings = torch.nn.functional.embedding(atom_number,
                                                         self.embedding_list)
         return atom_enbeddings
+
+class Highway(nn.Module):
+    """ Create a highway layer. y = H(x) * T(x) + x * (1 - T(x))
+
+    H(x) = activation_fn(matmul(W_H, x) + b_H) is the non-linear transformed output
+    T(x) = sigmoid(matmul(W_T, x) + b_T) is the transform gate
+
+    Implementation based on paper
```

Comment:
Please format this as a reference section https://numpydoc.readthedocs.io/en/latest/format.html#references